mindmeld/components/nlp.py

```diff
@@ -1077,12 +1099,15 @@ def process(
             (ProcessedQuery): A processed query object that contains the prediction results from \
                 applying the hierarchy of natural language processing models to the input text.
         """
+        del language
```

Comment:
probably we should add a TODO here so that we can clean up this API in a future version of MindMeld

Reply:
done